nft/collection/mint_process.go

```diff
@@ -246,16 +193,74 @@ func (opp *MintProcessor) PreProcess(
 		return nil, operation.NewBaseReasonError("invalid signing; %w", err)
 	}
 
+	opp.idxes = map[extensioncurrency.ContractID]uint64{}
+	opp.idxStates = map[extensioncurrency.ContractID]state.State{}
+	opp.boxes = map[extensioncurrency.ContractID]*NFTBox{}
+	opp.boxStates = map[extensioncurrency.ContractID]state.State{}
+	for i := range fact.items {
+		collection := fact.items[i].Collection()
+
+		if _, found := opp.idxes[collection]; !found {
+			if st, err := existsState(StateKeyCollection(collection), "design", getState); err != nil {
+				return nil, operation.NewBaseReasonError(err.Error())
+			} else if design, err := StateCollectionValue(st); err != nil {
+				return nil, operation.NewBaseReasonError(err.Error())
+			} else if !design.Active() {
+				return nil, operation.NewBaseReasonError("deactivated collection; %q", design.Symbol())
+			} else if policy, ok := design.Policy().(CollectionPolicy); !ok {
+				return nil, operation.NewBaseReasonError("policy of design is not collection-policy; %q", design.Symbol())
+			} else if whites := policy.Whites(); !design.Creator().Equal(fact.Sender()) {
```

Comment:
@wyuinche
This part seems to have been changed last time to check the mint qualification based on the whitelist list only.

Reply:
@soonkuk It's been fixed. 🙂

Reply:
@wyuinche 👍